flixel/text/FlxText.hx

```diff
@@ -35,11 +35,46 @@ import openfl.utils.AssetType;
 /**
  * Extends FlxSprite to support rendering text. Can tint, fade, rotate and scale just like a sprite. Doesn't really animate
  * though. Also does nice pixel-perfect centering on pixel fonts as long as they are only one-liners.
+ * 
+ * ## Autosizing
+ * 
+ * By default `FlxText` is fully autosize - its height and width are determined automatically
+ * by its text content. But you can control this behavior:
+ * 
+ * ```haxe
+ * // 1. fixed width and height
+ * var text = new FlxText(0, 0, 100, "Hello");
+ * text.fieldHeight = 50;
+ * // or
+ * text.fieldWidth = 100;
+ * text.fieldHeight = 50;
+ * 
+ * // 2. fixed width but auto height
+ * var text = new FlxText(0, 0, 100, "Hello");
+ * // or
+ * text.fieldWidth = 100;
+ * text.fieldHeight = -1;
+ * 
+ * // 3. autosize (by default)
+ * var text = new FlxText(0, 0, -1, "Hello");
+ * // or
+ * text.autoSize = true;
+ * text.wordWrap = false;
+ * // or
+ * text.fieldWidth = text.fieldHeight = -1;
+ * 
+ * // 4. auto width but fixed height
+ * // has no sense and works as default autosize
+ * ```
```

Comment:
I think this would be much clearer without the examples and they should just be in the doc for autoSize, fieldWidth and fieldHeight, maybe here it can have an autosizing section that refers the dev to those fields. i.e:

```
 * ## Autosizing
 * 
 * By default `FlxText` is autosized to fit it's text. To set a fixed size,
 * use the `fieldWidth`, `fieldHeight` and `autoSize` fields.
```

Reply:
Maybe add a snippet or update a text demo or even both? What do you think?

Reply:
some snippets for text would be awesome, but that would happen after this gets merged